kong/conf_loader/init.lua

```diff
@@ -551,6 +551,20 @@ local CONF_SENSITIVE = {
   pg_password = true,
   pg_ro_password = true,
   proxy_server = true, -- hide proxy server URL as it may contain credentials
+  cluster_cert = true, -- may contain absolute or base64 value of the the cert
```

Comment:
small style suggestion: 

could we align these same comments `-- may contain absolute or base64 value of the the key`?

Reply:
I don't feel `cluster_cert` is sensitive, it only contains the public key of the cert and is sent over the wire in cleartext as part of TLS handshake. Only `cluster_cert_key` is sensitive.

Reply:
Addressed